tests/test_helpers.py

```diff
@@ -305,19 +201,3 @@ def encode(result):
     assert isinstance(output.status_code, int)
     assert output.body == "{'data': {'result': None}}"
     assert output.status_code == 200
-
-
-def test_encode_execution_results_with_pretty():
-    execution_results = [ExecutionResult({"test": "Hello World"}, None)]
-
-    output = encode_execution_results(execution_results, encode=json_encode_pretty)
-    body = output.body
-    assert body == "{\n" '  "data": {\n' '    "test": "Hello World"\n' "  }\n" "}"
-
-
-def test_encode_execution_results_not_pretty_by_default():
```

Comment:
Why were these tests removed?

Reply:
I merged `json_encode_pretty` into `json_encode` with a extra argument that enables the pretty output. But you are right, I should have changed the function to enable the pretty output and test it.

Reply:
Note that `json_encode_pretty` is part of the official API listed in the README. You should change the README and add an alias for backward compatibility if you remove it.

Reply:
@Cito Good catch!

Reply:
I went with the alias and placed a note on the readme explaining that. Also, reincluded the two pretty tests.